.idea/.gitignore

```diff
@@ -1,6 +1,6 @@
 # Default ignored files
```

Comment:
lets place this file into root folder 
also, .idea folder should be removed from merge request